app/views/articles/_article_links.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2 class="related-links__heading"><%= title %></h2>
+
+<div class="related-links">
+  <ul class="unstyled-list">
+    <% article_links.each do |article_link| %>
+      <li class="related-links__link">
+        <%= render article_link %>
+        <% if article_link.date.present? %>
+          <div>
+            <%= article_link.date.strftime("#{article_link.date.day.ordinalize} %B %Y") %>
```

Comment:
I don't know the code standards of the frontend project, but I think here you can use the I18n.l and move the format to a locale.